source/chapters/microphones.rst

```diff
@@ -411,3 +411,28 @@ input.
 
 DJ controllers typically do not supply phantom power required for condenser
 microphones. Stand-alone audio interfaces typically do supply phantom power.
+
+Microphone Ducking
+------------------
+Microphone ducking is a technique to improve the audibility of speech by reducing the volume of background music while a voice signal is recorded by the microphone and allows listeners to hear the voice more clearly and consistently.
+While using Mixxx with a microphone, you might want the music to automatically fade into the background each time you speak, or you might prefer lowering the volume of the music to a specific level. Microphone ducking helps you achieve this.  The music will 'duck' out of the way of the microphone audio.
+
+.. figure:: ../_static/2.2/microphone-ducking.png
+   :align: center
+   :width: 40%
+   :figwidth: 90%
+   :alt: The microphone ducking section
+   :figclass: pretty-figures
+
+   The microphone ducking controls
+
+The :guilabel:`Strength` knob adjusts the microphone talkover ducking strength and its behaviour depends on any of the Microphone talkover modes described below.
+Clicking  on the :guilabel:`Auto` button will switch it to any of three modes:
+
+   * :guilabel:`Off`: When you speak through the microphone, the music  volume does not reduce.
+   * :guilabel:`Auto`: When you speak through the microphone, the music volume will be reduced when Mixxx detects your voice. You can adjust the amount by which the volume is automatically reduced with the Strength knob.
+   * :guilabel:`Manual`:In this mode, the music will play at the volume set by the Strength knob whenever the microphone Talk button is active, regardless of the volume coming through the microphone input. When you speak through your microphone, set the volume to which the music should reduce by turning the Strength knob to a level that suits you. When you disable Talk mode, the music will return to the regular volume
+
+.. hint:: To set the Ducking volume correctly, you can use manual mode to adjust the knob to your taste and then set to auto when happy.
+
+.. Note:: Enabling the Microphone Gain option will cause the ducking effect to be ineffective, so, you can disable it for the ducking effect to work correctly.
```

Comment:
@deborahtrez  where did this note come from? When you say "ineffective" do you mean "less effective", or "not effective at all?"

Reply:
When I enabled the Microphone Gain, and used microphone ducking, I noticed that the two would not work together (at all). Even with very low and very high ducking Strengths. The Gain option had to be off/disabled for the ducking to work.

Reply:
Is this note worthy?  Perhaps i should leave it out?

Reply:
Yes that seems like a bug.  Leave it out and then this is LGTM

Reply:
Please file a bug report on Launchpad then.

Reply:
> The Gain option had to be off/disabled for the ducking to wor

Are you talking about the tiny Gain 'mute' button in Deere 2.2?
When the input is muted there's no reason for the Ducking to kick in, so I'd say this is intended behaviour.

Reply:
Those gain buttons were removed for Mixxx 2.3 as they only cause confusion and we added PLAY buttons to Aux decks as equivalent for mics' TALK buttons for a consistent UI.

Reply:
I have filed the report [here](https://bugs.launchpad.net/mixxx/+bug/1903754) on the launchpad

Reply:
Marked as Invalid because the source of confusion is that the GAIN label should describe the Gain knob only but is right next to the Mute button.
So you discovered a UX issue instead :)

Reply:
Repairing the UI in 2.2 would be the right fix, but I doubt the effort is worth it since we're rather close to publishing 2.3.

Reply:
Okay. Let me remove this note section and commit.